lib/i18n/backend/simple.rb

```diff
@@ -44,7 +44,7 @@ def store_translations(locale, data, options = {})
         def available_locales
           init_translations unless initialized?
           translations.inject([]) do |locales, (locale, data)|
-            locales << locale unless (data.keys - [:i18n]).empty?
+            locales << locale unless data.size <= 1 && (data.empty? || data.has_key?(:i18n))
```

Comment:
Please replace with `if` condition to make it more readable: `if data.size > 1 || !(data.empty? || data.has_key?(:i18n))`

Reply:
I'd be happy to change it, but I'm not sure that improves readability. My mental model of this block is a reject we are implementing with `inject` (so that we get easy `locale, data` pairs) and that maps best to unless.

Does an i18n maintainer have a preference?

Reply:
Hi :) I am an i18n maintainer with an opinion on this topic.

I think `unless` is fine. I think the intention is clear here.